irohad/main/server_runner.cpp

```diff
@@ -20,6 +22,10 @@ ServerRunner::ServerRunner(const std::string &address,
       reuse_(reuse),
       tls_keypair_(tls_keypair) {}
 
+ServerRunner::~ServerRunner() {
+  shutdown(std::chrono::system_clock::now());
```

Comment:
It looks like we can use some const from the beginning of the times: `0` or `1`.